public/chore.js

```diff
@@ -30,4 +30,22 @@ choreList.forEach(function (chore) {
 });
 
 const app = document.querySelector('#app');
-app.appendChild(list);
\ No newline at end of file
+app.appendChild(list);
+
+const newChore = {
```

Comment:
new chore not needed to be created twice.

Reply:
Moved this code into the function and forgot to delete it.